lib/content-services/src/lib/dialogs/download-zip/download-zip.dialog.stories.ts

```diff
@@ -15,30 +15,26 @@
  * limitations under the License.
  */
 
-import { Meta, moduleMetadata, Story } from '@storybook/angular';
+import { applicationConfig, Meta, moduleMetadata, StoryFn } from '@storybook/angular';
 import { AlfrescoApiService } from '@alfresco/adf-core';
 import { MatButtonModule } from '@angular/material/button';
 import { DownloadZipDialogStorybookComponent } from './download-zip.dialog.stories.component';
-import {
-    AlfrescoApiServiceMock,
-    ContentApiMock,
-    DownloadZipMockService,
-    NodesApiMock
-} from './mock/download-zip-service.mock';
-import { DownloadZipDialogModule } from './download-zip.dialog.module';
+import { AlfrescoApiServiceMock, ContentApiMock, DownloadZipMockService, NodesApiMock } from './mock/download-zip-service.mock';
 import { DownloadZipService } from './services/download-zip.service';
 import { ContentService } from '../../common/services/content.service';
 import { NodesApiService } from '../../common/services/nodes-api.service';
+import { MatDialogModule } from '@angular/material/dialog';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { importProvidersFrom } from '@angular/core';
+import { CoreStoryModule } from '../../../../../core/src/public-api';
```

Comment:
something wrong with this import path, the story module should not be compiled together with the core lib